src/helpers/augmentedReality/ARSupportingDevice.js

```diff
@@ -0,0 +1,11 @@
+import { osVersion } from 'expo-device';
+import { useContext } from 'react';
+
+import { device } from '../../config';
+import { SettingsContext } from '../../SettingsProvider';
+
+export const ARSupportingDevice = () => {
```

Comment:
i think this method should be seen as a stand alone funktion `isARSupported` that returns `true` or `false`. why do you decide to have a component-like name `ARSupportingDevice` returning an object?

it would be easier to import: `..., isARSupported } from '../helpers';`, right? and you would not need to to `const { isARSupported } = ARSupportingDevice();` as `isARSupported()` is directly available.

Reply:
Yes, very true. I'll do it.